Changelog Nov 7 2024 (#396)
CloudCannon/platform-documentation: changelogs/2024-11-07_end-of-support-for-legacy-app.mdx

---
_schema: default
title: End of support for Legacy app
---

> The CloudCannon Legacy app is no longer available. Thank you to everyone who provided feedback during this process; we have addressed all issues brought to our attention. If you have any further feedback or questions, please don't hesitate to contact our friendly [support team](/suppport/).

Reviewer: ```suggestion
The CloudCannon Legacy app is no longer available. Thank you to everyone who provided feedback during this process; we have addressed all issues brought to our attention. If you have any further feedback or questions, please don't hesitate to contact our friendly [support team](/support/).
```

> This release added ARIA attributes to the password strength label, new key descriptions for the *Collections* section in the CloudCannon configuration file, removed the *Recent activity* section for users without permission to read it, and added a Setup icon if you delete your configuration file.

Reviewer: ```suggestion
This release added ARIA attributes to the password strength label, new key descriptions for the *Collections* section in the CloudCannon configuration file, removed the *Recent activity* section for users without permission to read it, and added an extra setup prompt if you delete your configuration file.
```

## Features & Improvements

* Improved accessibility by adding ARIA attributes to the password strength label under the *Password* field.
* Updated the description text for all keys under the *Collections* heading when you open the CloudCannon configuration file in the Data Editor.
* CloudCannon will no longer show the *Recent activity* section on the *Site Dashboard* if you do not have the `site:activity:read` permission.
* CloudCannon now prompts you to regenerate your CloudCannon configuration file if you delete it. The wand icon will reappear next to *Status* in your *Site Navigation* and the *Status* page will allow you to regenerate your CloudCannon configuration file.

## Fixes

* Fixed an issue where CloudCannon would temporarily display a second copy of a file after renaming a file.
* Fixed an issue where CloudCannon would incorrectly display a "Waiting for build..." message on Sites in Headless Mode.
* Fixed an issue where users on Client Sharing could not see the Site icon and previews for Bookshop components.
* Fixed an issue where users on Client Sharing could not log in to a Site if it did not have a CloudCannon configuration file.

> * Fixed an issue where Schemas did not automatically appear in the *\+ Add* button dropdown in the *Collection browser*.

Reviewer: ```suggestion
* Fixed an issue where Schemas did not appear in the *\+ Add* button dropdown in the *Collection browser*, if `add_options` was configured as an empty array.
```
